### PR TITLE
Adding new template+design, adding request sending&handling code, code refactoring.

### PR DESCRIPTION
**Pull request by rebeccashill**
Merge old website with new website. Add various jQuery and CSS files

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,185 +1,1224 @@
...
+                    <a id="closecontentpop">X</a>
+                </div>
+                <div class="blogtcontent">> Rebecca's documents</div>
+            </div>
+            <div class="contentwrap"></div>
+           
+            <div id="contentright">
+
+            <div class="post"><!-- post 1 ----------->
+            <!--<div class="postdesci">
```
Comment (sakshi-23) [inline portfolio/src/main/webapp/index.html]: Why is this block commented out?

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+    /**
```
Comment (sakshi-23) [inline portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java]: Please add text in the comment block

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,185 +1,1224 @@
...
+                </div><!--folders-->
+                
+                </div><!--contentleft-->
+                
+                
+                </div><!-- contentpop -->
+
+
+                 
+                <div id="sticky1" class="ui-draggable">
```
Comment (sakshi-23) [inline portfolio/src/main/webapp/index.html:420]: I see a lot of content on this page. Are you considering personalizing it or leaving it as is?

**Review by sakshi-23 — COMMENTED** [review]
Ideally this pull request should be broken down into 3 requests:
1. Adding the new template & design
2. Adding the request sending & handling code
3. Code cleanup and refactoring.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (rebeccashill) [reply portfolio/src/main/webapp/index.html:420]: Yes, I want to make all the icons draggable and personalize the stickys

**Review by sakshi-23 — APPROVED** [review]
Great work on integrating with the template!

**Comment by rebeccashill** [comment]
Last merge broke the live website :( Any idea how to fix? Seems like it's not fully registering the CSS

**Comment by sakshi-23** [comment]
Please can you link to the broken page. Might be easier to check what is
going wrong.

Thanks
Sakshi

**Comment by rebeccashill** [comment]
It seems to be working now...I didn't change anything so I guess appengine just took a minute to generate the full page? Honestly not too sure why

Either way website is working fine now!

**Comment by sakshi-23** [comment]
That is great! Though yeah that is a common issue with web apps. Sometimes
you might need to just restart or wait.

**Pull request merged**